src/v/container/include/container/fragmented_vector.h

```diff
@@ -308,6 +324,21 @@ class fragmented_vector {
 
         iter() = default;
 
+        /**
+         * Conversion operator allowing const_iterator to be converted to
+         * iterator.
```

Comment:
nit: isn't this allowing an `iterator` to be converted to a `const_iterator`?

Reply:
Good catch, fixed!